src/styles/table.scss

```diff
@@ -137,6 +140,39 @@ $block: #{$fd-namespace}-table;
 
   /****** Elements ******/
 
+  &__empty {
+    &-table-message {
+      padding: 1rem;
+      text-align: center;
+      display: flex;
+      align-items: center;
+      justify-content: center;
```

Comment:
There's a mixin you can use for this block of code. Please check `fd-flex-center` mixin.

Reply:
done